WordPress/Tests/WP/CronIntervalUnitTest.inc

```diff
@@ -160,3 +160,45 @@ add_filter( 'cron_schedules', function ( $schedules ) {
 	];
 	return $schedules;
 } ); // Error: 8 min.
+
+// Correctly handle fully qualified WP time constants.
+class FQNConstants {
+	public function __construct() {
+		add_filter( 'cron_schedules', array( $this, 'add_weekly_schedule' ) );
+		add_filter( 'cron_schedules', array( $this, 'add_eight_minute_schedule' ) ); // Error: 8 min.
```

Comment:
This comment suggests it would be an "Error", but there's nothing in the `getErrorList()` array?

Maybe this one was not meant to produce an error anymore?

Reply:
Throughout the test case file, it uses `// Error`. This is tied to the history of the sniff.

* This sniff was originally part of the set of `VIP` sniffs. At that time, the "interval too short" notice was an error.
* PR #1242, and in particular commit 52faccb08145e050e983cc123a97bc17f5a8a199, moved the sniff from the `VIP` category to the `WP` category (as one of the sniff we wanted to keep widely available when the `VIP` category would be removed). The preceding discussion on this can be found in #1157 and was also related to #1309.
* The [next commit](https://github.com/WordPress/WordPress-Coding-Standards/pull/1242/commits/77e908624be1ccd99057e9af70c3ece2f5e3c374) (part of the same discussion and PR) then changed the notice from an `error` to a `warning`, but didn't update the inline comments in the test case file.

I agree that it should say `Warning` in the test case file now, but that is outside the scope of this PR, so I have opened PR #2074 to address this for the pre-existing tests and will fix it up for the new tests in this PR.